docs/pages/components/contextual-menu.md

```diff
@@ -59,7 +59,7 @@ Implementation Guidelines:
                 <li><a href="#" class="fd-menu__item">Option 3</a></li>
             </ul>
             <div class="fd-menu__group">
-                <h1 class="fd-menu__title">Group Header</h1>
+                <h3 class="fd-menu__title">Group Header</h3>
```

Comment:
So I went back and forth on this, as contextual menu is made up of a "menu" component, I only put the warnings on menu because I thought it might be confusing having them on all the examples. Either way is fine with me though.

Reply:
Yeah, I guess this one is fine not having it.  It's not really a component anyway.